tests/output-module-formats/subtests/es-modules/parcel/index.md

```diff
@@ -1,14 +1,11 @@
 ---
-result: fail
+result: pass
 issue:
   - url: https://github.com/parcel-bundler/website/issues/631
     status: closed
   - url: https://github.com/parcel-bundler/website/issues/630
+    status: closed
   - url: https://github.com/parcel-bundler/parcel/issues/4890
```

Comment:
This bug is still open, which makes me think this should be a `partial`. WDYT, @devongovett?

Reply:
Hm, I worry that some sort of client-side routing could run into the “two separate instances of the same module” problem, but I also can’t come up with a non-contrived example. I’ll tag @jakearchibald to take a look at this once he’s  back from leave, but for now we can move on with this as a `pass` I‘d say.